### ISSUE TITLE
Gamecube Specific Controller Support

### ISSUE THREAD

**Issue by NintendoFan** [nipkownix/re4_tweaks#204]
### Describe the feature you'd like to see added

I want to be able to play the game the same way I played it the first time I ever did - With a Gamecube controller. This is technically already possible, but, of course the button prompts are all messed up and don't show correctly. There was a related suggestion in #105 but it seemed that slipped through the cracks, so I'm making a new issue here in hopes that it gets noticed and replied to.

Basically, replace the button prompts with the related Gamecube buttons if using a Gamecube controller, and maybe some other stuff? It's probably feasible to reimplement the original control structure of the original release, right?
